### PR TITLE
Feature/update baseline

### PR DESCRIPTION
**Pull request by iamdanfox**
I found it super difficult to dev against DCR in Eclipse because the autoformatter and checkstyle conflicted.  This meant that every time I saved a file, it broke checkstyle in Eclipse.

I've pulled in the most recent version of Baseline and (after disabling a few checks), ensured the entire codebase passes (I did make a few small tweaks to whitespace, but hopefully nothing too offensive).

This will help me both eclipse and intellij users dev against this codebase without tearing their hair out.

Open question: is it acceptable to use the baseline import ordering? It has changed a lot of files, but this is a one off hit.  I can figure out how to revert if necessary.

### CODE REVIEW

**Comment by felixdesouza** [comment]
Works inside IntelliJ!

**Comment by iamdanfox** [comment]
@CRogers thoughts?

File: src/test/java/com/palantir/docker/compose/execution/RetryerShould.java [hunk 1]
```diff
@@ -28,36 +23,39 @@
...
     }
 
     @Test
-    public void retry_the_operation_if_it_failed_once_and_return_the_result_of_the_next_successful_call() throws IOException, InterruptedException {
+    public void retry_the_operation_if_it_failed_once_and_return_the_result_of_the_next_successful_call() throws Exception {
         when(operation.call()).thenAnswer(MockitoMultiAnswer.<String>of(
-            firstInvocation  -> { throw new DockerComposeExecutionException(); },
-            secondInvocation -> "hola"
+                firstInvocation  -> { throw new DockerComposeExecutionException(); },
+                secondInvocation -> "hola"
```
Comment (CRogers) [inline src/test/java/com/palantir/docker/compose/execution/RetryerShould.java]: ewww, 8 character continuation indent

**Comment by CRogers** [comment]
Does this automatically set up import order in eclipse/intellij?

**Comment by j-baker** [comment]
yeah, baseline does that

On Sat, 23 Apr 2016 at 20:45 Callum Rogers <notifications@github.com<mailto:notifications@github.com>> wrote:

Does this automatically set up import order in eclipse/intellij?

**Comment by iamdanfox** [comment]
Yep, that was the biggest annoyance!

**Pull request merged**